packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardForm.js

```diff
@@ -16,15 +16,23 @@ const LoginCardForm = ({
   forgotPassword,
   rememberMe,
   submitError,
-  showError
+  showError,
+  attributes
```

Comment:
we usually just use `...props` and pass those along. no need to set a propType for this.